web/src/ImpactDetailMode.svelte

```diff
@@ -5,6 +5,7 @@
   import { SplitComponent } from "svelte-utils/top_bar_layout";
   import { DotMarker, gjPosition, layerId, Link, PrevNext } from "./common";
   import { ModalFilterLayer } from "./layers";
+  import EditableIntersectionLayer from "./layers/EditableIntersectionLayer.svelte";
```

Comment:
This is a read-only mode; the user shouldn't be able to click intersections and do anything here. Is this here to visualize the diagonal filters? If so, it looks like that's set up to grab data from `RenderNeighbourhoodOutput`. But this is a map-wide mode, so instead should we fold this into the `renderModalFilters()` WASM call or make one for diagonals?